rest-api-spec/src/main/resources/rest-api-spec/test/search/330_fetch_fields.yml

```diff
@@ -169,3 +169,51 @@ setup:
 
   - match: { hits.hits.0.fields.integer.0: 42 }
   - is_false: hits.hits.1.fields.integer
+
+---
+"Test disable _source loading":
+  - do:
+      indices.create:
+        index:  test
+        body:
+          settings:
+            number_of_shards: 1
+          mappings:
+            properties:
+              keyword:
+                type: keyword
+              integer:
+                type: integer
+                store: true
+
+  - do:
+      index:
+        index:  test
+        id:     1
+        body:
+          keyword: "x"
+          integer: 42
+
+  - do:
```

Comment:
I tend to stick `refresh` on the index request.

Reply:
👍  this is nicer, especially when there's only one index request.